inc/command/class-command.php

```diff
@@ -487,6 +518,58 @@ protected function run_command( InputInterface $input, OutputInterface $output,
 		return $return_val;
 	}
 
+	/**
+	 * Bootstrap tests folder.
+	 *
+	 * @param string $tests_folder Target folder to boostrap the tests into.
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 *
+	 * @return void
+	 */
+	protected function bootstrap_codecept( string $tests_folder, InputInterface $input, OutputInterface $output ) {
+		$default_suites = $this->get_test_suites( 'vendor/altis/dev-tools/tests' );
+
+		$suites = $input->getArguments()['options'][1] ?? '';
+		if ( 0 === strpos( $suites, '-' ) ) {
+			$suites = $default_suites;
+		} else {
+			$suites = explode( ',', $suites );
+			$invalid = array_diff( $suites, $default_suites );
+			if ( count( $invalid ) ) {
+				throw new Exception(
+					sprintf(
+						'Invalid suites selected: "%s", available suites are: "%s".',
+						implode( ',', $invalid ),
+						implode( ',', $default_suites )
+					)
+				);
+			}
+		}
+
+		$base_path = getcwd(); // Do we have another way to detect this ?
+		$tests_folder = $base_path . '/' . $tests_folder;
+
+		if ( ! file_exists( $tests_folder ) ) {
+			mkdir( $tests_folder );
+		} else {
+			foreach ( $suites as $suite ) {
+				if ( file_exists( $tests_folder . '/' . $suite ) ) {
```

Comment:
Because this part of the command is running on the host OS might need to use `DIRECTORY_SEPARATOR` instead, similar in a few other places too. Not sure if PHP just handles it or not on Windows.

Reply:
PHP filesystem functions will always convert slashes to the appropriate directory separator automatically, so we don't really need to use `DIRECTORY_SEPARATOR` while compiling paths, maybe only for deconstructing paths coming from a system function, eg: exploding a path via the directory separator.

Reply:
I thought it did something like that but couldn't remember for sure. Nice 👍